pkg/scheduler/objects/queue.go

```diff
@@ -473,6 +473,12 @@ func (sq *Queue) GetPartitionQueueDAOInfo() dao.PartitionQueueDAOInfo {
 	} else {
 		queueInfo.Parent = sq.QueuePath[:strings.LastIndex(sq.QueuePath, configs.DOT)]
 	}
+	queueInfo.MaxRunningApps = sq.maxRunningApps
+	queueInfo.RunningApps = sq.runningApps
+	queueInfo.AllocatingAcceptedApps = make(map[string]bool)
```

Comment:
we can use a simple slice of strings here, we do not need the booleans:
```
queueInfo.AllocatingAcceptedApps = make([]string, len(sq.allocatingAcceptedApps))
```

Reply:
Good point, updated!